CommunicationSeminar/settings.py

```diff
@@ -104,7 +104,7 @@
             'PORT': '3306',
             'NAME': 'CommunicationSeminarDjango',
             'USER': 'root',
-            'PASSWORD': '2017%ComSem',
+            'PASSWORD': 'tempcbsql',
```

Comment:
I put a little message about how local development settings on slack. If you merge from develop and then create a dev settings file, you should be able to have this change locally without including it in your PR.
Let me know if that makes sense 🙂